src/utils/axios.ts

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import { getUserFromLocalStorage } from "./localStorage";
+
+// NOTE: Please use your own server port number, and will change to production server later
+export const axiosClient = axios.create({
+  baseURL: "http://localhost:8000/api/v1",
```

Comment:
Awesome. Here it could be modified to get from the `.env` file, which would make it easier to configure.